src/lib/components/layout/CTA/CTA.scss

```diff
@@ -1,52 +1,52 @@
-@import '../../../styles/tools/breakpoints';
-@import '../../../styles/tools/spacing';
+@use '../../../styles/tools/breakpoint';
+@use '../../../styles/tools/spacing';
 
-$_cta-breakpoint: lg;
-$_cta-spacing: spacing(1);
-$_cta-column-gap: spacing(3);
+$_breakpoint: lg;
```

Comment:
Thanks to SASS modules scoping we don't need globally unique names of private variables as they are now really private (things starting with `_` or `-` are not shared upwards).